dot/network/host.go

```diff
@@ -136,7 +140,7 @@ func newHost(ctx context.Context, cfg *Config) (*host, error) {
 					ok = append(ok, addr)
 				}
 			}
-			return ok
+			return append(ok, externalAddr)
```

Comment:
If `pubip.Get` function fails then `externalAddr` will be empty, will be a better approach to only appending it if it is not empty?

Reply:
good point I'll update it to append only if there's no error